docs/standard/base-types/best-practices-strings.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Best Practices for Using Strings in .NET"
-description: Learn how to use strings effectively in .NET applications.
+title: "Best Practices for Comparing Strings in .NET"
+description: Learn how to compare strings effectively in .NET applications.
```

Comment:
Since 95% of this file was about comparing strings, I think it's much clearer what it's about to say "best practices for comparing strings" rather than "best practices for using strings", which seems pretty vague to me.